internal/cri/server/restart.go

```diff
@@ -107,7 +107,11 @@ func (c *criService) recover(ctx context.Context) error {
 		metadata := sandboxstore.Metadata{}
 		err := sbx.GetExtension(podsandbox.MetadataKey, &metadata)
 		if err != nil {
-			return fmt.Errorf("failed to get metadata for stored sandbox %q: %w", sbx.ID, err)
+			log.G(ctx).
+				WithError(err).
+				WithField("sandbox", sbx.ID).
+				Warn("failed to get metadata for stored sandbox")
+			continue
```

Comment:
should probably also remove sbx from the sandbox store.. 

@mxpv @samuelkarp @fuweid thoughts?

Reply:
> should probably also remove sbx from the sandbox store..

Yes, there isn't much we can do about it without metadata.
I'd also do `Error` here instead of `Warn`.